specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
In terms of UI, we could provide our tabulation information as a result of them uploading the CVRs, and share that publicly and with the county and sos for them to check against the announced results.  At some point, the public has to know the tabulation the tool is relying on anyway.
These checks are all part of the AXIOM TBDs in the "CVR Export Verification" section that I commented on long ago. At least I think I did.....